core/src/i18n/en.yaml

```diff
@@ -514,12 +514,23 @@ dnsentries:
     ttl: TTL
     dns-name: DNS Name
     text: Text
+    target: Target
     targets: Targets
+    use-cname: Use CNAME
+  name_singular: DNS Entry
   placeholders:
-    targets: Target records (CNAME or A records), one per line
-    text: Text records, one per line
     ttl: Enter the time to live.
-    dns-name: Enter DNS Name
+    dns-name: Enter the DNS Name.
+    target-a: Enter the A record Target.
+    target-cname: Enter the CNAME record Target.
+  tooltips:
+    use-cname: Switch to CNAME record.
+    use-a: Switch to A record.
+    target: Target must be an A record, or a CNAME record.
+    targets: Targets records, either target or text are required.
+    text: Text record must be a string.
+    texts: Text records, either target or text are required.
```

Comment:
```suggestion
    texts: Text records, either target or text, are required.
```

Reply:
I think the comma changes the meaning of this sentence a little, suggesting that text records are required, and they can be either target or text type (at least that's how I would understand it). 
What I meant here is that this is a text record input field, but in general, there are 2 input fields, one with target records, one with text records, and at least one of those fields is required. See the screenshot below.
<img width="1078" alt="Screen Shot 2021-10-14 at 09 46 05" src="https://user-images.githubusercontent.com/40663133/137273876-b58324e8-ab3f-4a4e-aabb-05c83b2bbda8.png">

Reply:
How about 

```suggestion
    texts: Either target or text are required.
```

or 

```suggestion
    texts: You must provide either target or text.
```